app/controllers/item.js

```diff
@@ -1,76 +1,91 @@
-import Ember from 'ember';
+import Ember from "ember";
 import config from "../config/environment";
 
 export default Ember.Controller.extend({
-
-  messageBox:     Ember.inject.service(),
-  application:    Ember.inject.controller(),
-  queryParams:    ['categoryId', 'sortBy'],
-  categoryId:     null,
-  cart:           Ember.inject.service(),
-  sortBy:         "createdAt",
-  item:           Ember.computed.alias('model'),
-  noNextItem:     Ember.computed.empty('nextItem'),
-  noPreviousItem: Ember.computed.empty('previousItem'),
-  hideThumbnails: Ember.computed.gt('item.sortedImages.length', 1),
-  smallScreenPreviewUrl: Ember.computed.alias('item.displayImage.smallScreenPreviewImageUrl'),
+  messageBox: Ember.inject.service(),
+  application: Ember.inject.controller(),
+  queryParams: ["categoryId", "sortBy"],
+  categoryId: null,
+  cart: Ember.inject.service(),
+  sortBy: "createdAt",
+  item: Ember.computed.alias("model"),
+  noNextItem: Ember.computed.empty("nextItem"),
+  noPreviousItem: Ember.computed.empty("previousItem"),
+  hideThumbnails: Ember.computed.gt("item.sortedImages.length", 1),
+  smallScreenPreviewUrl: Ember.computed.alias(
+    "item.displayImage.smallScreenPreviewImageUrl"
+  ),
   itemNotAvailableShown: false,
-  hasCartItems: Ember.computed.alias('application.hasCartItems'),
+  hasCartItems: Ember.computed.alias("application.hasCartItems"),
   isMobileApp: config.cordova.enabled,
 
   direction: null,
 
-  hasQuantityAndIsAvailable: Ember.observer('item.isAvailable', 'item.packages.@each.orderId', 'item.isUnavailableAndDesignated', function() {
-    var currentPath = this.get('target').currentPath;
-    var item = this.get("item");
-    var isItemUnavailable = this.get('item.isUnavailableAndDesignated');
-    if((currentPath === 'item' || currentPath === "package_category") && isItemUnavailable && isItemUnavailable !== null && !this.get("itemNotAvailableShown")) {
-      this.set("itemNotAvailableShown", true);
-      if(this.get('cart').hasCartItem(item)) {
-        this.get('cart').removeItem(item);
+  hasQuantityAndIsAvailable: Ember.observer(
+    "item.isAvailable",
+    "item.packages.@each.orderId",
+    "item.isUnavailableAndDesignated",
+    function() {
+      var currentPath = this.get("target").currentPath;
+      var item = this.get("item");
+      var isItemUnavailable = this.get("item.isUnavailableAndDesignated");
+      if (
+        (currentPath === "item" || currentPath === "package_category") &&
+        isItemUnavailable &&
+        isItemUnavailable !== null &&
```

Comment:
`isItemUnavailable && isItemUnavailable !== null` can be replaced with `isItemUnavailable` 

I think all those changes are from prettier, but it caught my eye